package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-jw-player",
-  "version": "1.12.0",
+  "version": "1.12.2",
```

Comment:
You can go ahead and leave out the version changes, we will handle that as we merge things into the repo.

Reply:
Sorry, didn't think that would be part of the pull request.